tests/sentry/tasks/test_daily_summary.py

```diff
@@ -72,9 +69,7 @@ def store_event_and_outcomes(
                     data["release"] = release
 
                 event = self.store_event(
-                    data=data,
-                    project_id=project_id,
-                    assert_no_errors=False,
+                    data=data, project_id=project_id, assert_no_errors=False, event_type="error"
```

Comment:
I can go through and update other tests that use `DEFAULT_EVENT_DATA` to create error events but for now I wanted to try this on something innocuous.